school/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] kindly add @rentals as it is the class used to set the relationship between person and book